spring/spring/src/test/resources/org/infinispan/spring/provider/sample/CachingBookDaoContextTest.xml

```diff
@@ -1,37 +0,0 @@
-<beans xmlns="http://www.springframework.org/schema/beans" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
They were used by CachingBookDaoContextTest.java which was removed in 47372df . I also ran the spring tests without them and everything passed.